app/components/dashboard/DcSunburstChart.vue

```
@@ -0,0 +1,598 @@
+        } finally {
+          // フィルター更新中フラグを非同期で下げる
+          setTimeout(() => {
+            isUpdatingFilters = false;
+          }, 10);
+        }
+      });
+    }
+
+    // フィルタの設定
+    if (filters.value.length > 0) {
+      isProgrammaticUpdate = true;
```

ESLint

[error] Extra semicolon.

(@stylistic/semi)